m2e-maven-runtime/org.eclipse.m2e.maven.runtime/src/main/java/org/eclipse/m2e/internal/maven/listener/M2EMavenBuildDataBridge.java

```diff
@@ -144,9 +146,9 @@ public static final class MavenProjectBuildData {
 			this.groupId = Objects.requireNonNull(data.get("groupId"));
 			this.artifactId = Objects.requireNonNull(data.get("artifactId"));
 			this.version = Objects.requireNonNull(data.get("version"));
-			this.projectBasedir = Path.of(data.get("basedir"));
-			this.projectFile = Path.of(data.get("file"));
-			this.projectBuildDirectory = Path.of(data.get("build.directory"));
+			this.projectBasedir = Paths.get(data.get("basedir"));
```

Comment:
Don't know why the CI build did not catch this, but the service version of this Plugin should be incremented by +1.

Reply:
Done in https://github.com/eclipse-m2e/m2e-core/pull/1086/commits/9d7df73a0f124a25669c9e13ef35ca4c07132dee.